README.md

```diff
@@ -108,12 +107,24 @@ project's `.sbt` file:
 addCompilerPlugin("com.softwaremill.clippy" %% "plugin" % "0.4.0" classifier "bundle")
 ````
 
-If you are using `scalac` directly, add the following option:
+If you are using `scalac` directly, add the following options as needed for your use:
 
 ````scala
 -Xplugin:clippy-plugin_2.11-0.4.0-bundle.jar
 ````
 
+If you are writing tests for clippy advice using `scalac` directly:
```

Comment:
I also added some docs for this option in my PR. It's a bit more detailed, so we can skip it here.